Danki2/Assets/Scripts/Abilities/InstantCast/Swipe.cs

```diff
@@ -0,0 +1,54 @@
+﻿using UnityEngine;
+
+[Ability(AbilityReference.Swipe)]
+public class Swipe : InstantCast
+{
+    private const float DashDuration = 0.2f;
+    private const float DashSpeedMultiplier = 1.5f;
+    private const float PauseDuration = 0.3f;
+    private const float DamageRadius = 3f;
+
+    public Swipe(Actor owner, AbilityData abilityData, string[] availableBonuses) : base(owner, abilityData, availableBonuses)
+    {
+    }
+
+    public override void Cast(Vector3 floorTargetPosition, Vector3 offsetTargetPosition)
+    {
+        Owner.MovementManager.LookAt(floorTargetPosition);
+
+        Owner.MovementManager.TryLockMovement(MovementLockType.Dash, DashDuration, Owner.StatsManager.Get(Stat.Speed) * DashSpeedMultiplier, Owner.transform.forward, Owner.transform.forward);
```

Comment:
The length of this line is a bit nuts.
```suggestion
        Owner.MovementManager.TryLockMovement(
            MovementLockType.Dash,
            DashDuration,
            Owner.StatsManager.Get(Stat.Speed) * DashSpeedMultiplier,
            Owner.transform.forward,
            Owner.transform.forward
        );
```

Reply:
Good point as usual fourhorsemen-master, breaking it down